README.md

```diff
@@ -1,259 +1,41 @@
-# learningOrchestra client package
+<p align="center">
+    <img src="./learningOrchestra-python-client.png">
+</p>
 
-## Installation
-Ensure which you have the python 3 installed in your machine and run:
-```
-pip install learning_orchestra_cliet
-```
-
-## Documentation
+# learningOrchestra-python-client
 
-After downloading the package, import all classes:
+Python client for [learningOrchestra](https://github.com/learningOrchestra/learningOrchestra).
 
-```python
-from learning_orchestra_client import *
-```
+# Installation
 
-Create a Context object passing a ip from your cluster in constructor 
-parameter:
+Requires Python 3.x
 
-```python
-cluster_ip = "34.95.222.197"
-Context(cluster_ip)
 ```
-
-After create a Context object, you will able to usage learningOrchestra, each 
-learningOrchestra functionality is contained in your own class, therefore, to 
-use a specific functionality, after you instantiate and configure Context 
-class, you need instantiate and call the method class of interest, in below, 
-there are all class and each class methods, also have an example of workflow 
-using this package in a python code.
-
-## DatabaseApi
-
-### read_resume_files
-
-```python
-read_resume_files(pretty_response=True)
+pip install learning-orchestra-client
```

Comment:
In this line you did change the package name from learning_orchestra_client to learning-orchestra-client, I have thinked which you want change the package name, or you change accidentally?

Reply:
I checked pypi and it was pip install learning-orchestra-client

https://pypi.org/project/learning-orchestra-client/

Reply:
It is true! I have configured to used "learning_orchestra_client", but pypi seems changed the deploy to learning-orchestra-client, i tested the 2 ways and both works, but i don't know if installing whit a name way and using with another will work.

Reply:
I think we will stick to "-" instead of using "_" because it is more conventional. So I will merge the PR.